test/import/import-k8s-util.test.ts

```diff
@@ -51,4 +52,127 @@ test('parseApiTypeName', () => {
     namespace: 'io.intstr',
     version: undefined,
   });
+});
+
+describe('safeParseJsonSchema', () => {
+
+  test('description is sanitized', () => {
+
+    const schema = {
+      definitions: {
+        MutatingWebhook: {
+          description: "*/console.log('hello')/*",
+          properties: {
+            sideEffects: {
+              description: 'some normal description',
+              type: 'string',
+            },
+          },
+          required: [
+            'sideEffects',
+          ],
+          type: 'object',
+        },
+      },
+    };
+    const parsed = safeParseJsonSchema(JSON.stringify(schema));
+    expect(parsed.definitions.MutatingWebhook.description).toEqual("_/console.log('hello')/*");
+  });
+
+  test('throws when a key is illegal', () => {
+
+    const schema = {
+      definitions: {
+        MutatingWebhook: {
+          description: "*/console.log('hello')/*",
+          properties: {
+            'sideEffects': {
+              description: 'some normal description',
+              type: 'string',
+            },
+            'not a word': {
+              type: 'string',
+            },
+          },
+          required: [
+            'sideEffects',
+          ],
+          type: 'object',
+        },
+      },
+    };
+    expect(() => safeParseJsonSchema(JSON.stringify(schema))).toThrow("Key 'not a word' contains non standard characters");
+  });
+
+  test('strips when value is illegal', () => {
+
+    const schema = {
+      definitions: {
+        MutatingWebhook: {
+          description: "*/console.log('hello')/*",
+          properties: {
+            sideEffects: {
+              description: 'some normal description',
+              type: 'string',
+              somethingElse: 'not a word',
+            },
+          },
+          required: [
+            'sideEffects',
+          ],
+          type: 'object',
+        },
+      },
+    };
+    const parsed = safeParseJsonSchema(JSON.stringify(schema));
+    expect(parsed.definitions.MutatingWebhook.properties.sideEffects.somethingElse).toEqual(SafeReviver.STRIPPED_VALUE);
```

Comment:
Not sure if I understand this test case, why is it stripped?

Reply:
Added a comment.